clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -80,6 +80,10 @@ public void replaceAtHead(long key, Chain expect, Chain update) {
   }
 
   public void put(long key, Chain chain) {
+    if (chain.isEmpty()) {
+      // holes may exist where empty chains may be replicated..just return in such cases
```

Comment:
Should a put of an empty chain map to a remove?  Seems odd to leave existing state in place?

Reply:
ok.